packages/core/core/src/Transformation.js

```diff
@@ -98,6 +97,15 @@ export default class Transformation {
     let results = await this.runPipeline(pipeline, asset);
     let assets = results.map(a => a.value);
 
+    for (let {request, result} of this.configRequests) {
+      //console.log('RESULT BEFORE', result);
+      let plugin =
+        request.plugin != null &&
+        (await this.parcelConfig.loadPlugin(request.plugin));
```

Comment:
To clarify, we are going to continue with `loadPlugin`, but we are moving away from rehydrate and reload, in a sense. Because we are going to serialize and deserialize it.

Reply:
Yeah, so before the config was being loaded first in the main process and then sent over to the worker, so plugins `loadConfig` would return a serialized result. This wouldn't be efficient for plugins that are run in the main process and it's not efficient or necessary now that we are are able to call `loadConfig` in workers. So now `loadConfig` returns a result that can directly be used by the plugin and Parcel will call `preSerializeConfig` and `postSerializeConfig` when appropriate.